drivers.json

```diff
@@ -28,6 +28,25 @@
       "Hardware-enforced Stack Protection Compatibility Test"
     ]
   },
+  "igb": {
+    "name": "Intel(R) 82576 Gigabit Dual Port Network Connection",
+    "device": "igb",
+    "type": 0,
+    "install_method": "no-drv",
+    "support": true,
+    "pretestcommands": [
+      {
+        "desc": "Rename igb ethernet adapter to SupportDevice0",
+        "run": "Rename-NetAdapter -Name (Get-NetAdapter -InterfaceDescription 'Intel(R) 82576 Gigabit Dual Port Network Connection').Name -NewName 'SupportDevice0'"
+      }
+    ],
+    "ignore_list": [
+      "NDISTest 6.5 - [2 Machine] - GlitchFreeDevice",
+      "NDISTest 6.5 - [2 Machine] - VMQReceiveQueueStateChecking",
+      "NDISTest 6.5 - [2 Machine] - VMQPowerManagement",
+      "NDISTest 6.0 - [1 Machine] - 1c_Mini6PerfSend"
```

Comment:
Does HLK2022 have Hardware-enforcement test for IGB? If yes, please add it.

Reply:
I don't know what is Hardware-enforcement test?

Reply:
This is a manual HCK test for Windows Server 2022 / Windows 11 for many device sets. All virtio devices have it.

Reply:
Done with 2497720967fd67d4dbdee1eb646d142364baa8e0.